cmd/plakar/subcommands/ptar/ptar_test.go

```diff
@@ -0,0 +1,73 @@
+package ptar
+
+import (
+	"bytes"
+	"os"
+	"path/filepath"
+	"testing"
+
+	_ "github.com/PlakarKorp/plakar/storage/backends/fs"
+	ptesting "github.com/PlakarKorp/plakar/testing"
+	"github.com/stretchr/testify/require"
+)
+
+func init() {
+	os.Setenv("TZ", "UTC")
+}
+
+func TestExecuteCmdPtarDefault(t *testing.T) {
+	bufOut := bytes.NewBuffer(nil)
+	bufErr := bytes.NewBuffer(nil)
```

Comment:
do we need these?  we never look at the output in this test, nor in the next one.

Reply:
you're right, removed in https://github.com/PlakarKorp/plakar/pull/859/commits/2074f3fa6694cb5f431b9af59ee0be46fd434fa4

Reply:
thank you!